cli/cli/commands/run/run.go

```diff
@@ -144,14 +143,6 @@ var StarlarkRunCmd = &engine_consuming_kurtosis_command.EngineConsumingKurtosisC
 			Type:    flags.FlagType_Bool,
 			Default: fullUuidFlagKeyDefault,
 		},
-		{
-			Key: mapPortsFlagKey,
```

Comment:
Remove this flag which disabled port forwarding in the Portal only.  This flag is not documented and not used at Kurtosis.  The new `no-connect` flag supersedes it.